workflow-controller/controller-service/src/main/java/io/hyscale/controller/util/ServiceProfileUtil.java

```diff
@@ -55,37 +61,52 @@
     }
 
     /**
-     * @param profilePath .../<profile-name>-<service-name>.hprof.yaml
-     * @return
+     * Gets service name from profile file.
+     * 1.returns service name if present with the key @HyscaleSpecFields#overrides.
+     * 2.else returns null when file is null or throws relative HyscaleException.
```

Comment:
Change this @HyscaleSpecFields#overrides. to {@link HyscaleSpecFields#overrides}.
*throws relevant HyscaleException
Even for the below method